vendor/init.go

```diff
@@ -66,22 +83,65 @@ var files = []file{
 	},
 }
 
+func addChrome() (err error) {
+	var chromeFile file
+	chromeFile.name = chromeFilename
+	chromeFile.browser = true
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx, option.WithHTTPClient(http.DefaultClient))
+	if err != nil {
+		return fmt.Errorf("Cannot create a storage client for downloading the chrome browser: %s", err)
+	}
+	BktHandle := client.Bucket(storageBktName)
+	lcFileObj := BktHandle.Object(lastChangeFile)
+	rc, err := lcFileObj.NewReader(ctx)
+	if err != nil {
+		return fmt.Errorf("Cannot create a reader for last_change file: %s", err)
```

Comment:
Use interpolation to replace last_change with the contents of lastChangeFile.

Actually, a better thing to do would be to create a GCS path:

gcsPath := fmt.Sprintf("gs://%s/%s", ...) 

and use that in error messages.